### DISCUSSION TITLE
如何用 SteamTools 1.8 解锁游戏或DLC？

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#9] [Question / Answer]
如何用 SteamTools 1.8 解锁游戏 或 DLC ？

**Comment by pjy612** [comment]
要让 tools 能解锁游戏或DLC，需要以下几个步骤：
* 如果你没有这个游戏
    1. 通过steamdb.info，定位游戏的appid 和 dlcid 列表。
    将列表批量框选后拖拽到 tools 悬浮窗。
    或将 appid 和 dlcid 存放到一个单独的txt文件中，将文件拖拽到 tools 悬浮窗。
    2. 通过 tools 重启 Steam。
    3. 1.8 目前 游戏解锁成功 的话， 重启steam后直接能在库存中看到。
    如果没有出现游戏，可以通过 steamdb app介绍页的 Install 按钮触发安装。
    也可以打开开始菜单-运行，输入 steam://install/appid 来触发安装。
    4. 由于只是只解锁游戏或DLC的游玩权限。
    所以并不会触发下载功能，会出现空包情况。
    请自行将游戏的本体和DLC内容放入游戏所在的目录内后，再进行游玩。

* 如果你库里有了这个游戏（家庭分享 或者 离线号下载
    1. 直接将游戏从库里拖到悬浮窗
    2. DLC可以参考没有游戏的情况，也可以在游戏商店里面 拖拽DLC的图标。
    当然 最好还是 弄清楚 这个DLC是否包含内容。

**特别说明！如果 dlcid 刚好和 app 中的 某个 depotid 相同时，可能会导致卡更新。
这种dlc 如果你密钥清单不全，不推荐通过tools解锁。  
请选择使用其他第三方dlc解锁插件，比如 creamapi**

**Reply by Mygodok** [reply]
老哥内容仍然处于加密状态怎么解决

**Reply by pjy612** [reply]
缺密钥

**Reply by mjlhcf** [reply]
> 缺密钥

嘤嘤嘤，没收到邮件，还缺哪一步么

**Reply by Mygodok** [reply]
> 缺密钥

已解决，我之前通过共享进的库，新添加的dlc失效应该是清单和密匙过期导致的
感谢回复

**Reply by pjy612** [reply]
> > 缺密钥
> 
> 嘤嘤嘤，没收到邮件，还缺哪一步么

未到

**Reply by Pongsakorn132** [reply]
content configuration unavailable when i download game on steam

**Reply by pjy612** [reply]
> content configuration unavailable when i download game on steam

That means you don't have an lastest manifest

**Reply by Pongsakorn132** [reply]
why ps5 joystick  non detech after run steamtools

**Reply by pjy612** [reply]
> why ps5 joystick non detech after run steamtools

Why ask me, I'm not the author of steamtools
